sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -168,6 +168,35 @@ private[sql] object JDBCRDD extends Logging {
   }
 
   /**
+   * Converts value to SQL expression.
+   */
+  private def compileValue(value: Any): Any = value match {
```

Comment:
I just moved `compileValue`, `escapeSql`, and `compileFilter` in this companion object area.